packages/react-router-dom/server.tsx

```diff
@@ -11,12 +11,11 @@ import {
   invariant,
   UNSAFE_convertRoutesToDataRoutes as convertRoutesToDataRoutes,
 } from "@remix-run/router";
-import type { Location, RouteObject, To } from "react-router-dom";
+import { Location, RouteObject, Routes, To } from "react-router-dom";
```

Comment:
mixing types and actual objects here, I think our linter should have complained about this?

Reply:
huh, yeah I would have thought so too - let me take a look.  We might only have the TS eslint stuff in the remix repo

Reply:
ok, added https://typescript-eslint.io/rules/consistent-type-imports/ in [97b4bd1](https://github.com/remix-run/react-router/pull/9227/commits/97b4bd12b43e7ce0072c907e4eba0902662e2023)